server/tools/docker-entrypoint.sh

```diff
@@ -22,6 +22,10 @@ if [ "$KEYCLOAK_HOSTNAME" != "" ]; then
     if [ "$KEYCLOAK_HTTPS_PORT" != "" ]; then
         SYS_PROPS+=" -Dkeycloak.hostname.fixed.httpsPort=$KEYCLOAK_HTTPS_PORT"
     fi
+
+    if [ "$KEYCLOAK_ALWAYS_HTTPS" != "" ]; then
+            SYS_PROPS+=" -Dkeycloak.hostname.fixed.alwaysHttps=$KEYCLOAK_ALWAYS_HTTPS"
```

Comment:
@stianst Shouldn't we always set that when using HTTPS?

Reply:
@slaskawi +1

Reply:
@slaskawi Not sure what you mean.

Reply:
Shouldn't we always add `-Dkeycloak.hostname.fixed.alwaysHttps=$KEYCLOAK_ALWAYS_HTTPS` if `KEYCLOAK_HTTPS_PORT` is not empty?

Reply:
No, because some folks may want both http and https. For instance they may have a reverse proxy that uses http, but internal clients use https.